CommandAPI/commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPICommand.java

```diff
@@ -328,29 +402,11 @@ public void setPermission(CommandPermission permission) {
 		return aliases;
 	}
 
-	/**
-	 * Sets the aliases for this command
-	 * @param aliases the aliases for this command
-	 */
-	public void setAliases(String[] aliases) {
-		this.aliases = aliases;
-	}
-
 	/**
 	 * Returns the requirements that must be satisfied to run this command
 	 * @return the requirements that must be satisfied to run this command
 	 */
-	public Predicate<CommandSender> getRequirements() {
-		return requirements;
-	}
-
-	/**
-	 * Sets the requirements that must be satisfied to run this command
-	 * @param requirements the requirements that must be satisfied to run this command
-	 */
-	public void setRequirements(Predicate<CommandSender> requirements) {
-		this.requirements = requirements;
-	}
+	public Predicate<CommandSender> getRequirements() { return requirements; }
```

Comment:
Inline methods is bad code style that doesn't fit with everything else in every other class in this project.

Reply:
Hm... I think that that was automatically done by my ide; must of slipped by-